### PR TITLE
chore(docs): Add documentation for email opt-in checkbox check/uncheck.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
Document:

* signup.checkbox.change.marketing-email-optin.checked
* signup.checkbox.change.marketing-email-optin.unchecked

issue #2660

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
@rfk & @TDA - wanna review?

File: docs/client-metrics.md [hunk 1]
```diff
@@ -200,6 +200,8 @@ The event stream is a log of events and the time they occurred while the user is
 * signup.password.hidden - password is hidden
 * signup.email-optin.visible.true - email opt-in is visible
 * signup.email-optin.visible.false - email opt-in is not visible
+* signup.checkbox.change.marketing-email-optin.checked - user checked the email opt-in
+* signup.checkbox.change.marketing-email-optin.unchecked - user unchecked the email opt-in
```
Comment (vladikoff) [inline docs/client-metrics.md]: is this missing `change` 

 ?

Comment (shane-tomlinson) [inline docs/client-metrics.md]: These were copied directly out of the `POST /metrics` endpoint, that normalization must occur elsewhere.

Comment (vladikoff) [inline docs/client-metrics.md]: Datadog does it

Comment (vladikoff) [inline docs/client-metrics.md]: I guess we should document what the content server sends, but that could make it harder to find in DataDog

Comment (shane-tomlinson) [inline docs/client-metrics.md]: We could also normalize to `_` on the content server, before POSTing to the `/metrics` endpoint. That would probably affect existing kibana queries though.

**Comment by TDA** [comment]
Looks good to me @shane-tomlinson

**Comment by rfk** [comment]
Merging so the docs are up to date; if we adjust to use "_" rather than "-" we'll need to apply it uniformly rather than in this PR.

**Pull request merged**